### ISSUE TITLE
Enahncement suggestion: Allow picking the preferred target enchantment for conflicts

### ISSUE THREAD

**Issue by hilari0n** [aviettran/minecraft-anvil-calc#51]
Currently there's no way to indicate which of the available enchantments we'd like to end up with on the item, while it could greatly impact the overall cost (and thus the optimal operation order). It would be great, if there would be a way to indicate, which of the (conflicting) enchantments we'd like to end up with, if the combinations of the available enchantments on the input items allow for different results.
E.g. if we have fished two books, one with Unbreaking III and Silk Touch and another one with Mending and Fortune III, we may want to apply both of them to a tool to get a tool with Silk Touch (and Unbreaking III and Mending) or a tool with Fortune III (and Unbreaking III and Mending) and the order of optimal operations would differ for those two scenarios. The current version of the calculator seems to always prefer the Silk Touch result (probably as it has a lower optimal cost, than the Fortune III path).

**Comment by aviettran** [comment]
This feature already exists in the "Preserve" checkbox next to each enchantment. Here is an example:

https://aviettran.github.io/minecraft-anvil-calc/?items=%5B%7B%22name%22%3A%22book%22%2C%22enchantments%22%3A%5B%7B%22name%22%3A%22unbreaking%22%2C%22level%22%3A3%7D%2C%7B%22name%22%3A%22silk_touch%22%2C%22level%22%3A1%7D%5D%2C%22index%22%3A0%2C%22penalty%22%3A0%7D%2C%7B%22name%22%3A%22book%22%2C%22enchantments%22%3A%5B%7B%22name%22%3A%22mending%22%2C%22level%22%3A1%7D%2C%7B%22name%22%3A%22fortune%22%2C%22level%22%3A3%2C%22preserve%22%3Atrue%7D%5D%2C%22index%22%3A1%2C%22penalty%22%3A0%7D%5D

**Comment by hilari0n** [comment]
> This feature already exists in the "Preserve" checkbox next to each enchantment. [...]

I have not noticed this! Thank you for pointing this out! (And thank you for this great tool!)

It seems there's a minor issue with how the link works. When I opened the link you provided, the Fortune III enchantment was indeed preserved, but the checkbox beside this enchantment was **not** visibly marked. Only when I marked it and unmarked it, I was able to get a correct result when marking Silk Touch for preserving (otherwise I was getting "No items or items cannot be combined.").
